.rubocop_todo.yml

```diff
@@ -483,11 +452,6 @@ Style/RedundantBegin:
 Style/RedundantParentheses:
   Enabled: false
 
-# Offense count: 505
-# Cop supports --auto-correct.
-Style/RedundantSelf:
```

Comment:
This one makes our code highly inconsistent IMO, and calling `.self` when it's not required makes our objects less open to change. i.e: "I want to move method X to class Y, now I also have to look for all `self.` instances that call the method"